waterdata/tests/commands/lookup_generation/test_commands.py

```diff
@@ -0,0 +1,51 @@
+"""
```

Comment:
I added these tests to increase coverage over the lookup generation commands. They just confirm that JSON files are generated when run with some mock RDB responses (defined in waterdata/tests/commands/lookup_generation/conftest.py).